lms/envs/common.py

```diff
@@ -2766,3 +2766,9 @@
     'options': {},
 }
 PROCTORING_SETTINGS = {}
+
+STUDENT_RESPONSES_REPORT_SUPPORTED_TYPES = set([
```

Comment:
Nit: You can actually create a `set` directly like:

```python
STUDENT_RESPONSES_REPORT_SUPPORTED_TYPES = {
    'problem',
    'submit-and-compare',
    'freetextresponse',
}
```

So basically like a list but with `{}` instead of `[]`.